ComputerInterface/QueueManager.cs

```diff
@@ -28,19 +25,19 @@ public static IQueueInfo GetQueue()
             var queueString = GetQueueString();
             return Queues.Find(x => x.QueueName == queueString);
         }
-        
+
         static string GetQueueString()
-		{
+        {
             string currentQueue = PlayerPrefs.GetString("currentQueue", "DEFAULT");
-            if (currentQueue != "DEFAULT" && currentQueue != "COMPETITIVE" && currentQueue != "MINIGAMES")
+            /* if (currentQueue != "DEFAULT" && currentQueue != "COMPETITIVE" && currentQueue != "MINIGAMES")
             {
                 PlayerPrefs.SetString("currentQueue", "DEFAULT");
                 PlayerPrefs.Save();
                 currentQueue = "DEFAULT";
-            }
+            }*/ // most likely the best idea as of right now just in case it's ignored again
```

Comment:
Was there any bugs this fixed? or just future proofing?

Reply:
I primarily did it mainly because the last time gamemodes were touched to add Minigames queue this part was forgotten + I'm pretty sure there's an entire interface just for adding custom queues here